jbehave-support-core/src/main/java/org/jbehavesupport/core/ssh/SshSteps.java

```diff
@@ -79,11 +97,32 @@ public void markStartTime(String startTimeAlias) {
         testContext.put(startTimeAlias, ZonedDateTime.now());
     }
 
+    @Given("log read start timestamp is set to now")
+    public void saveLogReadStartTime(){
+        logReadStartTime = ZonedDateTime.now();
+    }
+
+    @Given("log read end timestamp is set to now")
+    public void saveLogReadEndTime(){
+        logReadEndTime = ZonedDateTime.now().plusSeconds(1);
+    }
+
+    @Given("log read start timestamp is set to saved value [$contextAlias]")
+    public void setScenarioStartOnSaved(ExpressionEvaluatingParameter<String> contextAlias) {
+        logReadStartTime = ZonedDateTime.parse(contextAlias.getValue());
+    }
+
+    @Given("log read end timestamp is set to saved value [$contextAlias]")
+    public void setScenarioEndOnSaved(ExpressionEvaluatingParameter<String> contextAlias) {
+        logReadEndTime = ZonedDateTime.parse(contextAlias.getValue());
+    }
+
     @Then("the following data are present in [$systemQualifier] log:$presentData")
     public void logContainsData(String systemQualifier, String stringTable) {
         checkDataPresence(systemQualifier, scenarioStart, stringTable, containsVerifier);
     }
 
+    @Deprecated
```

Comment:
Can you also add some doc about why its deprecated and what to use instead?

Reply:
Added comment